front/app/components/admin/SurveyDropdownSettings/DropdownSettings.tsx

```diff
@@ -0,0 +1,191 @@
+import React from 'react';
+
+import {
```

Comment:
Same with this one - I just moved this from a container into the shared components/admin folder.